src/sql/platform/dialog/wizardNavigation.component.ts

```diff
@@ -0,0 +1,91 @@
+import { IBootstrapParams } from 'sql/services/bootstrap/bootstrapService';
+import { Event, Emitter } from 'vs/base/common/event';
+import { Wizard } from './dialogTypes';
+import { IWorkbenchThemeService } from 'vs/workbench/services/themes/common/workbenchThemeService';
+import { attachStyler } from 'vs/platform/theme/common/styler';
+import { SIDE_BAR_BACKGROUND } from 'vs/workbench/common/theme';
+
+export class WizardNavigationParams implements IBootstrapParams {
+	wizard: Wizard;
+	navigationHandler: (index: number) => void;
+}
+
+@Component({
+	selector: WizardNavigation.SELECTOR,
+	providers: [],
+	template: `
+		<div #container class="wizardNavigation-container">
+			<ng-container *ngFor="let item of _params.wizard.pages; let i = index">
+				<div class="wizardNavigation-pageNumber">
+					<div class="wizardNavigation-connector" [ngClass]="{'invisible': !hasTopConnector(i), 'active': isActive(i)}"></div>
+					<a [attr.href]="isActive(i) ? '' : null" [title]="item.title">
+						<span class="wizardNavigation-dot" [ngClass]="{'active': isActive(i), 'currentPage': isCurrentPage(i)}" (click)="navigate(i)">{{i+1}}</span>
+					</a>
+					<div class="wizardNavigation-connector" [ngClass]="{'invisible': !hasBottomConnector(i), 'active': isActive(i)}"></div>
+				</div>
+			</ng-container>
+		</div>
+	`
+})
+export class WizardNavigation implements AfterViewInit {
+	public static readonly SELECTOR = 'wizard-navigation';
+
+	private _onResize = new Emitter<void>();
+	public readonly onResize: Event<void> = this._onResize.event;
+
+	@ViewChild('container', { read: ElementRef }) private _container: ElementRef;
+	constructor(
+		@Inject(forwardRef(() => ElementRef)) private _el: ElementRef,
+		@Inject(forwardRef(() => ChangeDetectorRef)) private _changeRef: ChangeDetectorRef,
+		@Inject(IBootstrapParams) private _params: WizardNavigationParams,
+		@Inject(IWorkbenchThemeService) private _themeService: IWorkbenchThemeService) {
+	}
+
+	ngAfterViewInit() {
+		this._themeService.onThemeChange(() => this.style());
+		this.style();
+		this._params.wizard.onPageChanged(() => this._changeRef.detectChanges());
+	}
+
+	hasTopConnector(index: number): boolean {
+		return index > 0;
+	}
+
+	hasBottomConnector(index: number): boolean {
+		return index + 1 !== this._params.wizard.pages.length;
+	}
+
+	isActive(index: number): boolean {
+		return index <= this._params.wizard.currentPage;
+	}
+
+	isCurrentPage(index: number): boolean {
+		return index === this._params.wizard.currentPage;
+	}
+
+	navigate(index: number): void {
+		if (this.isActive(index)) {
+			this._params.navigationHandler(index);
+		}
+	}
+
+	private style(): void {
+		let theme = this._themeService.getTheme();
+		let navigationBackgroundColor = theme.getColor(SIDE_BAR_BACKGROUND);
+		if (theme.type === 'light') {
+			navigationBackgroundColor = navigationBackgroundColor.lighten(0.03);
+		} else if (theme.type === 'dark') {
```

Comment:
what about hc?

Reply:
hc is handled by the default setting of `SIDE_BAR_BACKGROUND` without changing the color at all